src/utils/serialize.ts

```diff
@@ -74,7 +74,7 @@ export const getCrispUserData = (
     partner_activated_at: partnerDetails.createdAt,
     feature_live_chat: updatePartnerAccessResponse.featureLiveChat,
     feature_therapy: updatePartnerAccessResponse.featureTherapy,
-    therapy_sessions_remaining: updatePartnerAccessResponse.therapySessionsRemaining,
-    therapy_sessions_redeemed: updatePartnerAccessResponse.therapySessionsRedeemed,
+    therapySessionsRemaining: updatePartnerAccessResponse.therapySessionsRemaining,
+    therapySessionsRedeemed: updatePartnerAccessResponse.therapySessionsRedeemed,
```

Comment:
Because of this change that was made

`await updateCrispProfile(partnerAccessUpdateDetails, client_email);`

Reply:
but it can still be named whatever we like, to send to crisp. we should use snakecase consistently

`languageDefault` -> `language_default`
`therapySessionsRemaining` -> `therapy_sessions_remaining`

Reply:
If we want Crisp to update the value of the key it has to be consistent or it will create a new record. When a user account is created we were using `therapy_sessions_remaining` but with the way the code has been updated to work with the db field name it has to be updated to `therapySessionsRedeemed`. It's either we update the name in the db or we update the name in crisp.

Reply:
ah gotcha, the data is being dynamically set and doesn't always go through a serializer where we can rename?

in that case it might be better to make _all_ crisp fields camelcase so at least its consistent when people go to use these fields

Reply:
Okay no worries - will refactor